.murdock

```diff
@@ -425,7 +471,9 @@ compile() {
 
     if [ ${should_check_kconfig_hash} != 0 ]; then
         if [ ${kconfig_test_hash} != ${test_hash} ]; then
+            tput setaf 1
```

Comment:
what does this do?

Reply:
Ahh this is just temporary, when I run in my console it shows this warning in red, easier to scan while things are going.  Please ignore.